v20.2/build-a-go-app-with-cockroachdb-upperdb.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Build a Go App with CockroachDB and upper/db
+summary: Learn how to use CockroachDB from a simple Go application with the upper/db data access tool.
+toc: true
+twitter: false
+---
+
+<div class="filters filters-big clearfix">
+    <a href="build-a-go-app-with-cockroachdb.html"><button class="filter-button">Use <strong>pgx</strong></button></a>
+    <a href="build-a-go-app-with-cockroachdb-pq.html"><button class="filter-button">Use <strong>pq</strong></button></a>
+    <a href="build-a-go-app-with-cockroachdb-gorm.html"><button class="filter-button">Use <strong>GORM</strong></button></a>
+    <a href="build-a-go-app-with-cockroachdb-upperdb.html"><button class="filter-button current">Use <strong>upper/db</strong></button></a>
+</div>
+
+This tutorial shows you how build a simple Go application with CockroachDB and the [upper/db](https://upper.io/) data access tool.
+
+## Before you begin
+
+{% include {{page.version.version}}/app/before-you-begin.md %}
+
+<section class="filter-content" markdown="1" data-scope="secure">
+
+## Step 1. Create the `maxroach` user and `bank` database
+
+{% include {{page.version.version}}/app/create-maxroach-user-and-bank-database.md %}
+
+## Step 2. Generate a certificate for the `maxroach` user
+
+Create a certificate and key for the `maxroach` user by running the following command:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach cert create-client maxroach --certs-dir=certs --ca-key=my-safe-directory/ca.key
+~~~
+
+The code samples will run with `maxroach` as the user.
+
+## Step 3. Create the `accounts` table
```

Comment:
@xiam is it possible for upper/db to create the `accounts` table in the code?

Reply:
Sure, we can integrate this step into the code

Reply:
@vy-ton I updated that bit in https://github.com/cockroachlabs/hello-world-go-upperdb/pull/2